lib/stepper.js

```diff
@@ -1,9 +1,10 @@
-var Board = require("../lib/board.js"),
-  events = require("events"),
-  util = require("util");
+var Board = require("../lib/board.js");
+var __ = require("../lib/fn.js");
+var events = require("events");
+var util = require("util");
 
-var priv = new Map(),
-  steppers = [];
+var priv = new Map();
+var steppers = new Map();
```

Comment:
If this is changed to a Map, then occurrences of `steppers` needs to be changed accordingly: https://github.com/Resseguie/johnny-five/blob/stepper-test/lib/stepper.js#L144-L145